0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml

```diff
@@ -10,7 +10,6 @@
     <History>
         The Parthenon was the most magnificent temple in all of Ancient Greece. It was built after the Greco-Persian Wars, replacing an older structure that was destroyed by the Persians. It was decorated with many sculptures, the biggest being the Athena Parthenose. Constructed by Pheidias, the Athena Parthenose was around 38 feet (11.5 meters) tall and plated with gold and ivory.
         Unfortunately, the Parthenon has not aged well. Over the years, the structure has been converted both into a Greek Orthodox church, a mosque, and an ammunition dump that exploded and destroyed the building's center. It still stands on the Athenian acropolis, but it is now a ruin, with most of its statues stolen or destroyed.
-    </History>
```

Comment:
I suppose you removed this closing tag by accident?

Reply:
Yeah, oops.